docs/cloud/faq/providers.md

```diff
@@ -0,0 +1,14 @@
+# Providers
+
+### Azure: Do you use Locally redundant storage (LRS) or Zone-redundant storage (ZRS) ?
```

Comment:
Typos:
```suggestion
### Azure: Do you use Locally Redundant Storage (LRS) or Zone Redundant Storage (ZRS)?
```

Reply:
is well Locally-redudant & Zone-redudant in azure doc

Reply:
See https://azure.microsoft.com/en-us/blog/azure-zone-redundant-storage-in-public-preview/.

Reply:
and the opposite in the docs ;-) 
https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy#zone-redundant-storage

Reply:
They use both, but I'd like to have it consistent/